dll/shellext/netplwiz/netplwiz.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:         ReactOS Disconnect Network Drive
+ * FILE:            dll/shellext/netplwiz/netplwiz.c
+ * PURPOSE:
+ *
+ * PROGRAMMERS:     Jared Smudde
+ */
```

Comment:
Please use the new file header format: https://www.reactos.org/wiki/Coding_Style#File_Structure